app/src/main/java/org/cru/godtools/ui/dashboard/tools/CategoryAdapter.kt

```diff
@@ -0,0 +1,42 @@
+package org.cru.godtools.ui.dashboard.tools
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.ObservableField
+import androidx.lifecycle.Observer
+import org.ccci.gto.android.common.recyclerview.adapter.SimpleDataBindingAdapter
+import org.cru.godtools.databinding.DashboardListItemCategoryBinding
+
+class CategoryAdapter() :
+    SimpleDataBindingAdapter<DashboardListItemCategoryBinding>(), Observer<List<String>> {
+
+    val callbacks = ObservableField<CategoryAdapterCallbacks>()
+    var selectedCategory = ObservableField<String>()
```

Comment:
instead of messing around with observing the live data to populate a `selectedCategory` field, you should just set the LiveData object directly here to directly set on the layout